packages/overdrive/lib/components/MinimalModal/MinimalModal.spec.jsx

```diff
@@ -0,0 +1,26 @@
+import { render } from '@testing-library/react';
+import * as React from 'react';
+
+import { MinimalModal } from './MinimalModal';
+
+describe('<MinimalModal />', () => {
+	const testBodyText =
+		'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis convallis neque a laoreet maximus. Vestibulum hendrerit quam at mi venenatis faucibus at vel nisi. In ut risus et ipsum tincidunt tempor. Suspendisse potenti. Praesent faucibus posuere risus, at congue mauris porttitor ut. Donec sit amet elit vitae purus dictum aliquet quis ut ligula. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Vestibulum dui sapien, porttitor ac erat vel, malesuada rutrum mauris. Nam arcu tellus, pretium ut aliquet eget, ultrices vel est. Maecenas dapibus volutpat eros a volutpat.';
+
+	it('should not throw', () =>
+		expect(() => render(<MinimalModal />)).not.toThrow());
+
+	it('should match snapshot without title and body', () => {
+		expect(render(<MinimalModal isOpen />).baseElement).toMatchSnapshot();
+	});
+
+	it('should match snapshot with body', () => {
+		expect(
+			render(
+				<MinimalModal isOpen>
+					<p>{testBodyText}</p>
+				</MinimalModal>,
+			).baseElement,
+		).toMatchSnapshot();
```

Comment:
I can see the new MinimalModal component in the StoryBook from Chromatic but it doesn't seem to show in the Chromatic component list/changes - is that expected?

Reply:
Good point @smccarthy94 never really paid much attention to what the change list looks like for new components but like you said the satorybook story has definitely been added so I'd be surprised if future changes would not be picked there

<img width="1680" alt="Screen Shot 2021-06-18 at 8 53 24 am" src="https://user-images.githubusercontent.com/1334204/122482320-a66c3f00-d013-11eb-9255-181df762da05.png">

Reply:
Yeah it definitely looks like future changes will be checked against the snapshot as expected. Was just curious of the behaviour between Chromatic/SB etc in this case :)

Reply:
Ok so @smccarthy94 turns out I was wrong and this was explicitly disabled for chromatic as was the Standard modal too. These being modals don't translate to anything meaningful in chromatic unfortunately. When I enabled those I just got the triggers captured and not the actual modals

Reply:
<img width="487" alt="Screen Shot 2021-06-18 at 9 25 30 am" src="https://user-images.githubusercontent.com/1334204/122484005-2516ab80-d017-11eb-9a2c-7017324460e6.png">